ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -285,6 +292,23 @@ export const SimulationDetails: React.FC<SimulationDetailsProps> = ({
     transactionId,
   });
 
+  const { trackInlineAlertClicked } = useAlertMetrics();
```

Comment:
Minor, would it help to isolate all this in a local `BalanceChangeAlert` component in this file?